src/components/Sidebar/components/SideBarItem/SideBarItem.tsx

```diff
@@ -1,11 +1,28 @@
 import React, { ReactNode } from "react";
-import "styled-components/macro";
+import styled from "styled-components/macro";
 import { Color } from "variables";
 
 type Props = {
   children: ReactNode;
 };
 
+const Progress = styled.div`
+  background-color: #24292e;
```

Comment:
Hi Kate, if you check #49 you can that I linked a color for the background. 
It can be found here: [Cinder](https://github.com/HandsOnLearning/the-project/blob/master/src/variables.ts#L2). 

While the one you choose is pretty close, for consistency sake it would be best to use that color like this:
```suggestion
  background-color: ${Color.Cinder};
```
(Color is already imported, see the top of the file)